resource-managers/kubernetes/core/pom.xml

```diff
@@ -20,7 +20,7 @@
   <parent>
     <groupId>org.apache.spark</groupId>
     <artifactId>spark-parent_2.11</artifactId>
-    <version>2.2.0-SNAPSHOT</version>
+    <version>2.1.0</version>
```

Comment:
I think so. This is particularly because we likely want to publish these libraries to a Maven repository as well - something we still need to discuss the specifics of. One example use case is for developing custom implementations of [DriverServiceManager](https://github.com/apache-spark-on-k8s/spark/blob/k8s-support-alternate-incremental/resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/rest/kubernetes/DriverServiceManager.scala) which would require projects to take a dependency on `spark-kubernetes`. But if we publish these we need to choose a version string on the pom files that differs from what Spark is already publishing to maven central.